.travis.yml

```diff
@@ -1,11 +1,21 @@
 language: node_js
 node_js:
-  - "5"
-  - "5.1"
-  - "4"
-  - "4.2"
-  - "4.1"
-  - "4.0"
-  - "iojs"
-script: "npm run coverage"
-after_script: "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js"
\ No newline at end of file
+- '5'
+- '5.1'
+- '4'
```

Comment:
Do we need all of them?

Reply:
I'm not sure TBH, but it was already like that so I left it :slightly_smiling_face:

Reply:
Uhh ok. It seems this is to test the code against multiple Node versions https://blog.travis-ci.com/first_class_nodejs_support_on_travis_ci 

So I'm fine with leaving it as is, since this was supposed to be a non-breaking compatible change.